README.md

```diff
@@ -1,45 +1,221 @@
+**Knowledge Asset is the new, AI‑ready resource for the Internet**
 
-We define **decentralized knowledge graph (DKG)** as a global shared knowledge graph that is designed to benefit organizations and individuals by providing a common infrastructure for data exchange. The DKG:
+Knowledge Assets are verifiable containers of structured knowledge that live on the OriginTrail DKG and provide:
+- **Discoverability - UAL is the new URL**. Consider Uniform Asset Locators (UALs) a kind of URL that identify a piece of knowledge and make it easy to find and connect with other Knowledge Assets.
+- **Ownership - NFTs enable ownership**. Each Knowledge Asset is created with an NFT token that enables trusted ownership and verifiability of your knowledge.
+- **Verifiability - On-chain information trail**. The blockchain tech increases trust, security, transparency, and the traceability of information.
 
-* Enables Dapps with search, integration, analytics, AI and ML capabilities for any data source: blockchains, IPFS, enterprise systems, web services, personal devices
-* Removes central authorities (decentralized infrastructure)
-* Enables permissionless PUBLISH and QUERY (public network)
-* Decentralized identity & Verifiable Credentials based access control (references private data)
+<br/>
 
+**Discover Knowledge Assets with DKG Explorer:**
+<div align="center">
+    <table>
+        <tr>
+            <td align="center">
+                <a href="https://dkg.origintrail.io/explore?ual=did:dkg:otp/0x5cac41237127f94c2d21dae0b14bfefa99880630/309100">
+                  <img src="images/knowledge-assets-graph1.svg" width="300" alt="Knowledge Assets Graph 1">
+                </a>
+                <br><b>Supply Chains</b>
+            </td>
+            <td align="center">
+                <a href="https://dkg.origintrail.io/explore?ual=did:dkg:otp/0x5cac41237127f94c2d21dae0b14bfefa99880630/309285">
+                  <img src="images/knowledge-assets-graph2.svg" width="300" alt="Knowledge Assets Graph 2">
+                </a>
+                <br><b>Construction</b>
+            </td>
+            <td align="center">
+                <a href="https://dkg.origintrail.io/explore?ual=did:dkg:otp/0x5cac41237127f94c2d21dae0b14bfefa99880630/309222">
+                  <img src="images/knowledge-assets-graph3.svg" width="300" alt="Knowledge Assets Graph 3">
+                </a>
+                <br><b>Life sciences and healthcare</b>
+            </td>
+            <td align="center">
+                <a href="https://dkg.origintrail.io/explore?ual=did:dkg:otp/0x5cac41237127f94c2d21dae0b14bfefa99880630/308028">
+                  <img src="images/knowledge-assets-graph4.svg" width="300" alt="Knowledge Assets Graph 3">
+                </a>
+                <br><b>Metaverse</b>
+            </td>
+        </tr>
+    </table>
+</div>
 
 
+</details>
 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
I would put less this back to top buttons, or generally remove them, not sure how much people use them